### PR TITLE
ps3netsrv: Update to  ps3netsrv 20240709

### PR DESCRIPTION
**Pull request by Hirador**
Update to ps3netsrv 20240709

## Description

Fixes # 

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Comment by Hirador** [comment]
@hgy59 This is a minor rev update, if anything else is needed please let me know

**Comment by hgy59** [comment]
@Hirador you must update cross/ps3netsrv too, otherwise nothing is updated.

**Comment by Hirador** [comment]
> @Hirador you must update cross/ps3netsrv too, otherwise nothing is updated.

Yes as i've said, this is a minor update, i've built everyting up before committing to GitHub (created an .spk for my architecture and tested it on my synology)

The version is still 1.47.46, only the revision changes so it won't affect PKG_VERS or any other variable in the Makefile inside cross/ps3netsrv, even the digests are not changing but the updates to the actual ps3netsrv are definitelly there.

**Comment by hgy59** [comment]
> The version is still 1.47.46, only the revision changes so it won't affect PKG_VERS or any other variable in the Makefile inside cross/ps3netsrv, even the digests are not changing but the updates to the actual ps3netsrv are definitelly there.

Sorry, but I do not understand.
cross/ps3netsrv is the only dependency that is included in the spk.
If there is no other dependency (is there anything downloaded while building cross/ps3netsrv?) the spk will be binary the same.

**Comment by hgy59** [comment]
> Other than that i have one question for you: is it possible to implement the field SPK_REV to accomodate letters?

No, this is not supported anymore since DSM 7 (see footnote <sup>9)</sup> in https://github.com/SynoCommunity/spksrc/issues/4524#issue-842607867)

**Comment by Hirador** [comment]
> > The version is still 1.47.46, only the revision changes so it won't affect PKG_VERS or any other variable in the Makefile inside cross/ps3netsrv, even the digests are not changing but the updates to the actual ps3netsrv are definitelly there.
> 
> Sorry, but I do not understand. cross/ps3netsrv is the only dependency that is included in the spk. If there is no other dependency (is there anything downloaded while building cross/ps3netsrv?) the spk will be binary the same.

I've made some research, i've actually found out where the problem is. 
first yes, the source.tar.gz of the specified version is downloaded when building cross/ps3netsrv, and here is where the actual problem lies. The developer has actually updated everything up, but by appling a revison logic he may have missed the overall update of the actual source (which is what i use to download the package). I've reached out the developer, i will proceed with the update once we've figured it out. 

Thanks for your patience

**Comment by Hirador** [comment]
@hgy59 @mreid-tt I've made some changes in the content fetching logic from the source since the developer made a standalone repo dedicated for ps3netsrv, if anything else is needed please let me know.

**Comment by Hirador** [comment]
@hgy59 waiting for you checks to proceed merging

File: spk/ps3netsrv/Makefile [hunk 1]
```diff
@@ -1,16 +1,15 @@
 SPK_NAME = ps3netsrv
-SPK_VERS = 1.47.46
-SPK_REV = 9
+SPK_VERS = 20240709
+SPK_REV = 10
 SPK_ICON = src/${SPK_NAME}.png
 
 DEPENDS = cross/$(SPK_NAME)
```
Comment (hgy59) [inline spk/ps3netsrv/Makefile:6]: Just a small change in preparation of #6255
The dependency list creation will not allow make variables in DEPENDS definitions

```suggestion
DEPENDS = cross/ps3netsrv
```

**Pull request merged**